types/jasmine/index.d.ts

```diff
@@ -982,25 +982,18 @@ declare namespace jasmine {
     }
 
     interface JsApiReporter extends CustomReporter {
+        new (): any;
+
         started: boolean;
         finished: boolean;
         runDetails: RunDetails;
 
-        new (): any;
-
-        suites(): {[id: string]: SuiteResult};
-        results(): any;
-    }
-
-    interface SuiteResult {
```

Comment:
In a follow up, it might be worth renaming CustomReporterResult to match Jasmine, which calls it SpecResult. No action required for now.

Reply:
One of my next PRs will be to update spec and suite properties. I can do it in that PR.